jf_agent/config_file_reader.py

```diff
@@ -24,6 +24,7 @@ class GitConfig:
     git_strip_text_content: bool
     git_redact_names_and_urls: bool
     gitlab_per_page_override: bool
+    git_verbose: bool
```

Comment:
For completeness we should probably add a mention of this into jf_agent/example.yml

Reply:
Good point!